lib/ssh_scan/banner.rb

```diff
@@ -29,6 +29,8 @@ def ssh_lib_guess()
         return SSHScan::SSHLib::CiscoSSH.new()
       when /ROS/i
         return SSHScan::SSHLib::ROSSSH.new()
+      when /DOPRASSH/i
```

Comment:
@agaurav77 yes, some of these initial examples were pulled from Shodan.  I'm sure there are tons more where that came from.  I just sampled 20-30 of them to flesh out other exception handling bugs and filed issues for them to add support.